examples/validate-fee/modules/shared_data/src/main/scala/com/my/validate_fee/shared_data/Validators.scala

```diff
@@ -0,0 +1,21 @@
+package com.my.validate_fee.shared_data
+
+import cats.Applicative
+import cats.syntax.all._
+import com.my.validate_fee.shared_data.errors.Errors.{UnexpectedAmount, valid}
+import com.my.validate_fee.shared_data.types.Types.{SampleUpdate, UpdateTypeOne, UpdateTypeTwo, updateTypeOneFee, updateTypeTwoFee}
+import org.tessellation.currency.dataApplication.dataApplication.DataApplicationValidationErrorOr
+import org.tessellation.schema.SnapshotOrdinal
+
+object Validators {
+  def validateSampleUpdateFee[F[_]: Applicative](unused: SnapshotOrdinal, update: SampleUpdate): F[DataApplicationValidationErrorOr[Unit]] =
+    update
+      .pure[F]
+      .map {
+        case UpdateTypeOne(name, fee) =>
+          UnexpectedAmount(name, updateTypeOneFee, fee.amount).unlessA(updateTypeOneFee === fee.amount)
```

Comment:
I think greater than or equal to would make a better comparison here.

Reply:
@AlexBrandes Can you clarify? Is it ok or an error if the given fee is less than expected?

Reply:
It's an error if the given fee is less than expected. It shouldn't be an error if the given fee is more than expected. If the cost to do some action is 10 and I pay 20, it's still enough and should be accepted.